packages/composite-checkout-wpcom/src/components/wp-contact-form.js

```diff
@@ -449,19 +449,31 @@ function ContactFormSummary() {
 		<GridRow>
 			<div>
 				<SummaryDetails>
-					<SummaryLine>
-						{ billing.firstName || '' } { billing.lastName || '' }
-					</SummaryLine>
-					<SummarySpacerLine>{ billing.email || '' }</SummarySpacerLine>
-					<SummaryLine>{ billing.address || '' } </SummaryLine>
-					<SummaryLine>
-						{ billing.city && billing.city + ', ' } { billing.state || billing.province || '' }
-					</SummaryLine>
-					<SummaryLine>
-						{ postalCode && postalCode + ', ' }
-						{ billing.country }
-					</SummaryLine>
+					{ billing.firstName ||
```

Comment:
I'm not sure I understand this logic. It seems like you're saying, "If we have the first name, just print that. Otherwise, print the first name and the last name inside a `SummaryLine`." Is that right?

Reply:
oh no, it looks like prettier did something funky there. I'm going to push an update. It should say if there's a first or last name, then print the `Summary` line.

Reply:
All cleaned up. Let me know how that looks.